src/components/elements/Inputs/Dropdown/Dropdown.tsx

```diff
@@ -159,16 +176,27 @@ const Dropdown = (props: PropsWithChildren<DropdownProps>) => {
                 props.className
               )}
             >
-              <span className="w-full line-clamp-1">
-                {formatOptionsList(options, toArray<any>(value)) || props.placeholder}
-              </span>
+              <div className="flex items-center gap-2">
+                <When condition={options && options.length && options.length > 0 && options[0].meta}>
```

Comment:
this can be simplified:
`condition={options?.[0]?.meta != null}`

Reply:
done